amical/tools.py

```diff
@@ -278,22 +278,61 @@ def cov2cor(cov):
     return cor, sigma
 
 
-def super_gaussian(x, sigma, m, amp=1, x0=0):
-    sigma = float(sigma)
-    m = float(m)
+def super_gaussian(
+    x: np.ndarray, window: float, m: float = 3.0, amp: float = 1.0, x0: float = 0.0
+) -> np.ndarray:
```

Comment:
Annotations are cool, but parameter names cannot be changed arbitrarily for keyword-allowed arguments (`sigma -> window`), since that would break user code calling the function with `sigma` passed as keyword.
```suggestion
def super_gaussian(
    x: np.ndarray, sigma: float, m: float = 3.0, amp: float = 1.0, x0: float = 0.0
) -> np.ndarray:
```

Reply:
I have changed the parameter name back to `sigma` 👍